src/components/SummaryRow.vue

```diff
@@ -81,7 +81,7 @@ export default {
     getKafkaState () {
       const vm = this
       vm.loading = true
-      let url = this.url + 'kafka_cluster_state'
```

Comment:
this change might be applicable in multiple places.

Reply:
It seems that the summary page is the only page fetching data from multiple clusters. this is probably why the url generation is slightly different to the other components. its the only site which we have issues.

Tried switching to the method the other components are using `this.$helpers.getURL('rebalance', params)` but the url was wrong. I think that summary is a special case here.